aslam_cv_cameras/include/aslam/cameras/ceres-jet-math.h

```diff
@@ -0,0 +1,50 @@
+#ifndef ASLAM_CAMERAS_CERES_JET_MATH_H_
+#define ASLAM_CAMERAS_CERES_JET_MATH_H_
+
+#include <cmath>
+
+namespace aslam {
+namespace jet_math {
+
+inline double sqrt(const double x) {
+  return std::sqrt(x);
+}
+
+inline double tan(const double x) {
+  return std::tan(x);
+}
+
+inline double atan(const double x) {
+  return std::atan(x);
+}
+
+// Mathematical functions for use with the Jet type of the Ceres solver.
+// Adapted from: ceres/jet.h
```

Comment:
what did you change? would it be possible to use the ceres include instead?

Reply:
tan and atan were already there, I just added the sqrt and moved it to an own header file. we do not have a dependency on ceres in aslam_cv, so including this header file is not possible atm